Mac-App/Mac-App/StateObserver/StateObserver.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  StateObserver.swift
+//  Mac-App
+//
+//  Created by Daniel GARCÍA on 03.05.20.
+//  Copyright © 2020 Acphut Werkstatt. All rights reserved.
+//
+
+import Combine
+import ReSwift
+
+final class StateObserver<State: StateType & Equatable>: StoreSubscriber {
```

Comment:
Has to be a class due to `StoreSubscriber` protocol constraints. 

Otherwise I think this could totally be a value type as it is not shared resource.